packages/frontend/src/features/scheduler/components/SchedulerModal2.tsx

```diff
@@ -1,22 +1,47 @@
-import { Code, Modal, Text } from '@mantine/core';
+import { Group, Modal, Text } from '@mantine/core';
+import { IconSend } from '@tabler/icons-react';
 import React, { FC } from 'react';
-import SchedulersModalContent from './SchedulerModalContent';
+import MantineIcon from '../../../components/common/MantineIcon';
+import SchedulerModalContent2 from './SchedulerModalContent2';
 
 // TODO: rename when replacement is complete
 const SchedulersModal2: FC<
-    Omit<React.ComponentProps<typeof SchedulersModalContent>, 'onClose'> & {
+    Omit<React.ComponentProps<typeof SchedulerModalContent2>, 'onClose'> & {
         name: string;
         onClose?: () => void;
     }
-> = ({ isOpen, onClose = () => {} }) => {
+> = ({
+    resourceUuid,
+    schedulersQuery,
+    createMutation,
+    isOpen,
+    isChart,
+    onClose = () => {},
+}) => {
     return (
-        <Modal opened={isOpen} onClose={onClose}>
-            <Text mb="xl">
-                This feature isn't ready yet. Paste this in your browser console
-                to use Scheduled Deliveries:
-            </Text>
-
-            <Code>localStorage.clear('scheduler-modal-2')</Code>
+        <Modal
+            opened={isOpen}
+            onClose={onClose}
+            size="lg"
+            title={
+                <Group spacing="xs">
+                    <MantineIcon icon={IconSend} size="lg" color="gray.7" />
+                    <Text fw={600}>Scheduled deliveries</Text>
+                </Group>
+            }
+            styles={(theme) => ({
+                header: { borderBottom: `1px solid ${theme.colors.gray[4]}` },
+                body: { backgroundColor: theme.colors.gray[2] },
```

Comment:
do we need the gray bg? for me it's too dark. assuming this is the bg:

![CleanShot 2023-10-05 at 17 48 09@2x](https://github.com/lightdash/lightdash/assets/962095/10633cdc-4371-4946-880a-4e82461d8e1f)

Reply:
Yea, I thought so too. I'll check with Priyanka.